src/js/config/Config.test.ts

```diff
@@ -2,7 +2,7 @@
 export default {
   rootUrl: "",
   analyticsKey: "39uhSEOoRHMw6cMR6st9tYXDbAL3JSaP",
-  defaultRefreshRate: 3000,
+  defaultRefreshRate: 5000,
```

Comment:
this makes the UI request some endpoints less frequent in integration/system-tests and should take some load of the CI-nodes which seem to have a hard time with those tests.